packages/x-components/src/components/result/base-result-image.vue

```diff
@@ -1,36 +1,30 @@
 <template>
   <picture ref="image" class="x-picture x-result-picture" data-test="result-picture">
-    <component
-      :is="animation"
-      v-if="!hasImageLoaded && !hasAllImagesFailed"
-      class="x-picture__image x-picture__image--placeholder"
-      data-test="result-picture-placeholder"
-    >
-      <!-- eslint-disable-next-line max-len -->
-      <!-- @slot (Required) Loading image content. It will be rendered while the real image is not loaded -->
-      <slot name="placeholder" />
-    </component>
-    <component :is="animation">
+    <img
+      v-if="!hasImageLoaded && imageSrc"
+      @error="flagImageAsFailed"
+      @load="flagImageLoaded"
+      loading="lazy"
+      :src="imageSrc"
+      :style="loaderStyles"
+      data-test="result-picture-loader"
+      alt=""
```

Comment:
I know that this is not going to be seen, but should we add a right alt?

Reply:
Trying to research a little bit into `alt` attributes, I found[ this awesome page](https://www.w3.org/WAI/tutorials/images/decision-tree/) explaining its use cases using a decision tree. In this case the image is not intended for the user, so we can use an empty alt. In the other case I was hesitant about using `alt="result.name"`. My concern is that the result name does not describe the image content. But as per that decision tree, when the image is used inside a link with no more text, which is the case that we currently have in our archetype, the alt can be used to identify the destination of the link, which is the result page, so we should be fine.

Reply:
So, should we make that change in this pr?